bootstrap/index.js

```diff
@@ -17,11 +17,19 @@ const withStatusCode = (execute) => (data, callback) => {
 const cacheOptions = envOptions()
 if (cacheOptions.enabled) logger.info('Cache enabled: ', cacheOptions)
 
+// Execution helper async => sync
+const _executeSync = (execute) => {
```

Comment:
This function prevents us from easily adding multiple middlewares in the future by dropping the resolved value, I would recommend instead reworking the `initHandlerXXX` functions and `withStatusCode` so that they are all async.

Reply:
Yes, we definitely need to rework `initHandlerXXX` functions and using async handlers from the start. Unfortunately, this is out of scope for this feature and will require a lot of changes through the repo. To do that we first need to bring in some changes from the private repo that are currently waiting.